docs/vsphere_ci/README.md

```diff
@@ -0,0 +1,173 @@
+# Building Windows VM images in vSphere
+
+## Goals
+This document focuses on building Windows images in vSphere 6.7 environment used in CI but can be used as example for
+other vSphere environments. We use Packer to automate installation and configuration of the Windows images. Packer 
+generates a vSphere VM template that can be converted to a VM. After the VM has been created, the machine-api can clone
+it multiple times as part of the Windows VM creation. The following steps need to be executed from the bastion instance
+or any instance that has access to the vSphere environent.
+
+## Installing Packer
+
+We assume the latest Packer has been installed. In case of our vSphere environment, we need to install Packer
+on the bastion host. The installation steps are:
+
+- Download [Packer](https://www.packer.io/downloads)
+    - `curl -o packer.zip https://releases.hashicorp.com/packer/1.6.6/packer_1.6.6_linux_amd64.zip`
+- Unzip using zip utility on the host
+    - `gunzip -S .zip packer.zip`
+- `chmod +x packer`
+- Update $PATH to include the Packer binary
+
+## Prerequsite files
+- Please ensure `scripts` directory is present in the location where you are running Packer from and has the
+following files:
+
+- autounattend.xml
+- install-vm-tools.cmd
+- powershell.ps1
+- authorized_keys
+- install-docker.ps1
+
+The `scripts/authorized_keys` file must be edited to contain a public key. The private key associated with the public key is
+what will be used by WMCO to configure VMs created from Windows VM. After deploying WMCO, this private key will be
+provided by the user in the form of a Secret.
+
+The `scripts/autounattend.xml` file must be edited to change the value of `WindowsPassword` to a user provided password.
+
+The provided autouninstall.xml
+- Installs VMWare tools
+- Runs powershell.ps1 script which configures SSH
+- Runs install-docker.ps1 which installs docker
+
+References:
+- https://github.com/guillermo-musumeci/packer-vsphere-iso-windows/blob/master/win2019.base/win2019.base.json
+- https://www.packer.io/guides/automatic-operating-system-installs/autounattend_windows
```

Comment:
Please don't throw links but rather have some text saying what these links refer to.